frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
+    BASE_URL = '/api';
```

Comment:
The base URL isn't needed for anything else than the fetch method so it could be just part of it. Also the case could be just a normal `baseUrl`.